parsing/language_extension.ml

```diff
@@ -87,13 +87,17 @@ let is_erasable : type a. a t -> bool = function
   | Module_strengthening | SIMD | Labeled_tuples | Small_numbers | Instances ->
     false
 
+let maturity_of_unique_for_drf = Alpha
+
+let maturity_of_unique_for_destruction = Alpha
```

Comment:
This is now unused; can we delete it and just have one `maturity_of_unique`?

Reply:
No, we later want to make `maturity_of_unique_for_drf` `Stable`, but want to keep `maturity_of_unique_for_destruction` as `Alpha`. That change has impacts on the production compiler though, so it shouldn't be done here.